source/guide_part-db.rst

```diff
@@ -0,0 +1,198 @@
+   - Installing snapappointments/bootstrap-select (v1.13.10): Extracting archive
+   - Installing squizlabs/php_codesniffer (3.4.2): Extracting archive
+   - Installing tecnickcom/tcpdf (dev-master ca4f9b8): Cloning ca4f9b86c9 from cache
+   0/12 [>---------------------------]   0%
+   8/12 [==================>---------]  66%
+  10/12 [=======================>----]  83%
+  11/12 [=========================>--]  91%
+  12/12 [============================] 100%Generating autoload files
+ [isabell@stardust html]$
+
+You have to create the Part-DB Database before configuring Part-DB like described here :manual_anchor:`additional <database-mysql.html#additional-databases>`:
+::
+
+ [isabell@stardust ~]$ mysql -e "CREATE DATABASE ${USER}_partdb" && echo "Created database ${USER}_partdb"
+ Created database isabell_partdb
+ [isabell@stardust ~]$
+
+
+Now point your browser to your uberspace URL and follow the instructions.
+
+You will need to enter the following information:
+
+  * an initial administrator password: Insert the credentials you want to use for the admin user while installing Part-DB
+  * your MySQL hostname, username and password: the hostname is ``localhost`` and you should know your MySQL :manual_anchor:`credentials <database-mysql.html#login-credentials>` by now. If you don't, start reading again at the top.
+  * your Part-DB database name from the previous step
+
+The installation wizard will prompt you to initialize the database and you will have to change the admin password afterwards.
+
+Backup
+======
+
+To create a backup of the installation you need the `data` folder and the database.
+To backup the `data` directory you copy it into your home:
+
+::
+
+ [isabell@stardust ~]$ cp --recursive ~/html/data ~/data
+ [isabell@stardust ~]$
+
+To create a backup of your database you dump it into a file:
+
+::
+
+ [isabell@stardust ~]$ mysqldump ${USER}_partdb > partdb_backup.sql
+ [isabell@stardust ~]$
+
+Restore
+=======
+
+You restore the database from the backup as follows:
+
+::
+
+ [isabell@stardust ~]$ mysql --database=${USER}_partdb < partdb_backup.sql
+ [isabell@stardust ~]$
+
+.. note::
+   If you get an error like "ERROR 1049 (42000): Unknown database 'test_partdb'" you have to create the databse as described in the installation instructions.
+
+The `data` directory can just be moved into the documentroot.
+
+
+Updates
+=======
+
+To update Part-DB you create a backup as described above or move the `data` folder away before replacing the installation:
+
+::
+
+ [isabell@stardust ~]$ mv ~/html/data ~/
+ [isabell@stardust ~]$
+
+Create an backup of your database:
+
+::
+
+ [isabell@stardust ~]$ mysqldump ${USER}_partdb > partdb_update_backup.sql
+ [isabell@stardust ~]$
+
+Remove the old installation:
+
+.. note::
+   If the trailing `/` is missing from the command below, the folder will be deleted and you will need to restore it using `mkdir /var/www/virtual/${USER}/html`.
```

Comment:
That's not true. The `html` directory will be deleted with or without slash.
```
[testt@stardust ~]$ ll /var/www/virtual/${USER}/html
-rwxr-xr-x. 1 testt testt 1171 Nov 22 17:41 nocontent.html

[testt@stardust ~]$ rm -rf /var/www/virtual/${USER}/html/
[testt@stardust ~]$ ll /var/www/virtual/${USER}/html
ls: cannot access /var/www/virtual/testt/html: No such file or directory

[testt@stardust ~]$ mkdir /var/www/virtual/${USER}/html

[testt@stardust ~]$ rm -rf /var/www/virtual/${USER}/html
[testt@stardust ~]$ ll /var/www/virtual/${USER}/html
ls: cannot access /var/www/virtual/testt/html: No such file or directory
```
Please have a look at the suggestion below and remove this note.